yeat/tests/test_cli.py

```diff
@@ -14,6 +14,8 @@
 from yeat.cli import InitAction, illumina
 from yeat.tests import data_file
 
+pytestmark = pytest.mark.short
+
```

Comment:
Curious: what is the purpose of this statement? Does it apply the mark to all tests in the file?

Reply:
> Does it apply the mark to all tests in the file?

Yes! It's a quick way to apply the mark decorator on all the functions in that file.